### ISSUE TITLE
`fnm use` fallback to default

### ISSUE THREAD

**Issue by jaredramirez** [Schniz/fnm#216]
Hey, thanks for the great package!

In my shell initialization, I'd like to call `fnm use` and have it use `.nvmrc` if it's there otherwise use the default version.

My use case is that when I open all of my various directories when working, my setup script opens `~/path/to/dir` directly, which doesn't trigger `--use-on-cd`. To address this, I've tried adding `fnm use` to my shell initialization. This works great, but if I open a new terminal window to `~`, it prints `No .nvmrc or .node-version file was found in the current directory. Please provide a version number.`. While this isn't the end of the world, it would be nice if `fnm use` would fallback to `default` if no version was specified.

My current work around is in my fish config is:
```fish
# ~/.config/fish/functions/load_fnm.fish
function load_fnm --on-variable="PWD"
    if test -e ./.nvmrc
        fnm use
    else
        fnm use default
    end
end

# ~/.config/fish/config.fish
fnm env --multi | source
load_fnm
```

**Comment by Schniz** [comment]
hmm. Why doesn't `use-on-cd` work? I would expect it to work. When I open a new shell in a specific directory on my zsh shell it loads the `.nvmrc` file in the current directory. I guess it isn't the case in fish? Maybe we can make it work the same way? (I'm not using Fish for a long time)

Can you look at the output from `fnm env --shell=fish` and let me know if there's something we can change?

**Comment by Schniz** [comment]
I mean, my workflow in my job is also to open lots of splits/shells. If it didn't change versions automatically I would be very annoyed, and I want it to work for others as well. So maybe I haven't understand the use case or problem.

If I haven't understand correctly, can you please record the issue with `asciinema`?

**Comment by jaredramirez** [comment]
You've understood it perfectly!

In reference to

> Can you look at the output from fnm env --shell=fish and let me know if there's something we can change?

After running `fnm env --shell=fish --use-on-cd`, I saw the fish function fnm uses (which is super close to the workaround I was using). From there, I copied it into my config directly and was able to reproduce the problem I was seeing. After defining the function `_fnm_autoload_hook`, we just have to call it to it'll set it for the current shell. This is just like the zsh cd function does [right here](https://github.com/Schniz/fnm/blob/master/executable/Env.re#L119).

I've submitted a super simple PR fixing this!
